Google.Api.Generator.Rest/Models/PackageModel.cs

```diff
@@ -384,15 +385,35 @@ int GetRevision()
 
             string GetApiDescription()
             {
+                // Unhelpful documentation links found by browsing Discovery docs.
+                string[] nonProductDocLinks =
+                {
+                    "https://cloud.google.com/",
+                    "https://cloud.google.com/learnmoreurl",
+                    "https://cloud.google.com/support/docs/apis",
+                    "https://www.google.com"
+                };
+
                 string upperCamelApiName = ApiName.ToUpperCamelCase(upperAfterDigit: false);
                 var prefix = _features.CloudPackageMap.TryGetValue(PackageName, out var cloudPackage)
                     ? $@"
       This is not the recommended package for working with {upperCamelApiName}, please use the {cloudPackage} package.
       This Google APIs Client Library for working with {upperCamelApiName} {ApiVersion} uses older code generation, and is harder to use."
                     : $"\n      Google APIs Client Library for working with {upperCamelApiName} {ApiVersion}.";
+                // Some packages have useful product documentation, but not all.
+                string productDocSection = _discoveryDoc.DocumentationLink is string productLink && !nonProductDocLinks.Contains(productLink) ? $@"
+
+      Product documentation is available at:
+      {productLink}" : "";
                 // The part of the package description that's the same for all packages - but can't be a constant due to the ApiName/ApiVersion part of the link.
                 string suffix = @$"
 
+      API reference documentation for this package is available at:
+      https://googleapis.dev/dotnet/{PackageName}/latest/api
```

Comment:
This seems to need to be:
`https://googleapis.dev/dotnet/{PackageName}/latest/api/{PackageName}.html`

Reply:
All fixed, thanks.